plane/source/docs/apms-failsafe-function.rst

```diff
@@ -173,20 +209,23 @@ condition persists for FS_LONG_TIMEOUT seconds.
    <td>1</td>
    <td>Circle/ReturnToLaunch</td>
    </tr>
+   <tr>
+   <td>2</td>
+   <td>FBWA</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>Disabled</td>
+   </tr>
    </tbody>
    </table>
 
-Long failsafe action (Plane:FS_LONG_ACTN)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Long failsafe action (:ref:`FS_LONG_ACTN<FS_LONG_ACTN>` )
+---------------------------------------------------------
+
+The action to take on a long (:ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` seconds) failsafe event. If the aircraft was in a stabilization or manual mode when failsafe started and a long failsafe occurs then it will change to RTL mode if :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` is 0 or 1, and will change to FBWA  and idle the throttle if :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` is set to 2.
 
-The action to take on a long (FS_LONG_TIMEOUT second) failsafe event in AUTO, GUIDED
-or LOITER modes. A long failsafe event in plane stabilization modes will
-always cause an RTL (ReturnToLaunch) or a QLAND or QRTL, dependent upon which Q_OPTION is selected), if in copter stabilization modes in a Quadplane. In AUTO modes you can choose whether it will RTL or continue with the mission. If FS_LONG_ACTN is 0
-then it will continue with the mission, if it is 1 then it will enter
-RTL mode. Note that if FS_SHORT_ACTN is 1, then the aircraft will
-enter CIRCLE mode after FS_SHORT_TIMEOUT seconds of failsafe, and will always enter
-RTL after FS_LONG_TIMEOUT seconds of failsafe, regardless of the FS_LONG_ACTN
-setting.
+If the aircraft was in an auto mode (such as AUTO or GUIDED) when the failsafe started then it will continue in the auto mode if :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` is set to 0, will change to RTL mode if :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` is set to 1 and will change to FBWA mode and idle the throttle if :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` is set to 2. If :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` is set to 3, the parachute will be deployed (make sure the chute is configured and enabled).
```

Comment:
QStab/QHover/QLoiter/QAcro/QAutotune will QRTL or QLAND depending on what the option bit is set to, the FS_LONG_ACTION has no impact on what it picks.

Reply:
fixed